rules.go

```diff
@@ -0,0 +1,11 @@
+package gorules
+
+import (
+	"github.com/quasilyte/go-ruleguard/dsl/fluent"
+)
+
+//nolint:deadcode,unused
```

Comment:
I propose adding `//nolint` to the line above `package gorules`. It will disable linting for this file completely.